submissions/kolawoleOgunfowokan/HTML/css/login.css

```diff
@@ -0,0 +1,109 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    /*background-color: #0919f7;*/
```

Comment:
The background-color in the comment is not the same as the one declared. Please adjust it accordingly.

Reply:
Done